SPEC.md

```diff
@@ -144,30 +144,30 @@ The key words "must", "must not", "required", "shall", "shall not", "should", "s
 
 #### CDI Devices
 
-The `cdiDevices` field describes the set of hardware devices that can be requested by the container runtime user.
+The `devices` field describes the set of hardware devices that can be requested by the container runtime user.
 Note: For a CDI file to be valid, at least one entry must be specified in this array.
 
-  * `cdiDevices` (array of objects, REQUIRED) list of devices provided by the vendor.
+  * `devices` (array of objects, REQUIRED) list of devices provided by the vendor.
     * `name` (string, REQUIRED), name of the device, can be used to refer to it when requesting a device.
       * Beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (\_), dots (.), and alphanumerics between.
       * e.g: `docker/podman run --device foo ...`
     * `nameShort` (array of strings, OPTIONAL), alternative names for the device. Can be used to reduce the CLI verbosity
       * Entries in the array MUST use the same schema as the entry for the `name` field
-    * `containerSpec` (object, OPTIONAL) this field is described in the next section.
+    * `ociEdits` (object, OPTIONAL) this field is described in the next section.
```

Comment:
I wonder if this spec should just specify what needs to happen to make a device available to a container.
OCI edits is the means the container runtime uses to satisfy this containerSpec.

i think the previous term `containerSpec` is more correct, as its notion is general and does not imply how a container runtime satisfies this spec.

we can say that when an OCI compliant runtime engine is used, this can be achieved through OCI edits.

Reply:
Will update back to containerSpec

Reply:
Edit: `ContainerEdits`